src/components/App/App.tsx

```diff
@@ -14,6 +15,7 @@ function App() {
       <Router>
         <Switch>
           <Route path="/" component={HomePage} exact />
+          <Route path="/place" component={PlaceInformation} />
```

Comment:
better to have route like path="/place/:id" to dynamicaly fetch place info in future, but can be later

Reply:
Agree, this is hardcoded for now, as name and address. I think picture should be something more bright and with coronavirus. I'll change. I pushed just for synchronization with you before went to sleep. Because of this bootstrap and badge class. To avoid dublication